src/components/Navbar/NavItem.js

```diff
@@ -0,0 +1,27 @@
+//Dependencies
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+//Components, assets, actions, styles etc..
+import { setActiveMenu } from '../../actions/index';
+
+const NavItem = (props) => {
+  const navState = useSelector((state) => state.nav);
+  const dispatch = useDispatch();
+
+  const setMenu = () => dispatch(setActiveMenu(props.name));
+  const closeMenu = () => dispatch(setActiveMenu(false));
+  const setClass = () => (props.name === 'Donate' ? 'donate' : 'nav');
+
+  return (
+    <div className={`${setClass()}--link`} onMouseEnter={setMenu} onMouseLeave={closeMenu}>
```

Comment:
I dont love dynamically setting classes like this for an loop what happens when donate is not last (some thing to think about)

Reply:
So the way it is currently written, the donate button would appear with the chunky purple styling regardless of where in the array it appears, even if it's not last.